rdt-open-reader/src/covidIndonesia/java/io/ona/rdt/fragment/CovidJsonFormFragment.java

```diff
@@ -0,0 +1,45 @@
+package io.ona.rdt.fragment;
+
+import android.os.Bundle;
+
+import com.vijay.jsonwizard.fragments.JsonFormFragment;
+import com.vijay.jsonwizard.presenters.JsonFormFragmentPresenter;
+
+import io.ona.rdt.interactor.RDTJsonFormInteractor;
+import io.ona.rdt.presenter.CovidJsonFormFragmentPresenter;
+
+import static io.ona.rdt.util.Constants.Encounter.RDT_TEST;
+import static io.ona.rdt.util.CovidConstants.Encounter.COVID_RDT_TEST;
+
+/**
+ * Created by Vincent Karuri on 13/07/2020
+ */
+public class CovidJsonFormFragment extends RDTJsonFormFragment {
+
+    @Override
+    protected boolean formHasSpecialNavigationRules(String eventType) {
+        return RDT_TEST.equals(eventType) || COVID_RDT_TEST.equals(eventType);
```

Comment:
No, there's no `in`-like operator. Isn't the set approach more verbose in achieving the same result?

Any advantage to doing it this way?

Reply:
Yea, some reasons that always apply:
- hashset inclusion will by ~O(1), a list of equality will be O(N)
- reduces code repetition, equals thing not repeated
- better semantic alignment, objectively what we're asking is are you in my group of interest?

Some reasons that sometimes apply
- if we expect our group of interest to statically change, it's less work (fewer char diff) to change code structured like this
- if we expect our group of interest to dynamically change, it's less work to replace it w/a function (that has a DB call or whatever)

Reply:
Cool, yea it's cleaner and faster as options are added.

Will create a util method instead in this case since we don't really need a global variable.

Reply:
Done.